#!/usr/bin//python3
import sys


def safe_print_integer_err(value):
    try:
        print("{:d}".format(value))
    except (ValueError, TypeError) as error_eror_error:
        stderr.write("Exception: {}\n".format(error_error_error))
        return False
    return True
